config/FindNDI.cmake

```diff
@@ -130,6 +130,8 @@ find_package_handle_standard_args( NDI
 mark_as_advanced( NDI_ROOT_DIR NDI_VERSION NDI_LIBRARY NDI_INCLUDE_DIR NDI_LIBRARY_DEBUG
     NDI_USE_PKGCONFIG NDI_CONFIG )
 
+message(FATAL_ERROR, ${NDI_VERSION})
```

Comment:
Instead of this, just add `NDI_VERSION` to the `REQUIRED_VARS` section of `find_package_handle_standard_args` above.

Reply:
Oops this is a debug message I forgot to remove

Reply:
I do think requiring `NDI_VERSION` is the right thing to do, though.